internal/pkg/skuba/deployments/ssh/assets/assets.go

```diff
@@ -40,6 +41,6 @@ EnvironmentFile=-/var/lib/kubelet/kubeadm-flags.env
 # This is a file that the user can use for overrides of the kubelet args as a last resort. Preferably, the user should use
 # the .NodeRegistration.KubeletExtraArgs object in the configuration files instead. KUBELET_EXTRA_ARGS should be sourced from this file.
 EnvironmentFile=-/etc/sysconfig/kubelet
-ExecStart=
+ExecStartPre=/bin/bash -c "findmnt -t bpf --mountpoint /sys/fs/bpf || mount bpffs /sys/fs/bpf -t bpf"
```

Comment:
Shouldn't we keep the `ExecStart=` and only add the `ExecStartPre`?

Reply:
Also, the CI seems very unhappy right now :)

Reply:
What's the point of having two `ExecStart=` lines in one unit? Only the later one, `ExecStart=/usr/bin/kubelet ...` should stay IMO, as it's the relevant one.

Reply:
Done